build.gradle.kts

```diff
@@ -64,9 +64,9 @@ subprojects {
 readme.readmeTemplate = file("docs/templates/README-TEMPLATE.md")
 
 ksciencePublish {
-    vcs("https://github.com/mipt-npm/kmath")
-    space(publish = true)
-    sonatype(publish = true)
+    github(rootProject.name, publish=false)
```

Comment:
Isn't it better to use an explicit name? It is not guaranteed to coincide with github name